Documentation/Installation.rst

```diff
@@ -54,5 +54,6 @@ Creating/Removing Network
 Configure MCVirt
 -----------------
 
-* The first time MCVirt is run, it creates a configuration file for itself, found in **/var/lib/mcvirt/config.json**.
-* Set the volume group for the VMs to be stored in, within the configuration file.
+* The first time MCVirt is run, it creates a configuration file for itself, found in **/var/lib/mcvirt/<Hostname>/config.json**.
+* Set ``vm_storage_vg`` in the configuration file to the name of volume group for the VMs to be stored in.
+* Set ``cluster_ip`` in the configuration file to your IP address.

```

Comment:
Migtn be worth saying what this field means, i.e. the IP address of the node that MCVirt clustering/DRBD communications will be performed on. Do you think it's important for this to be set on all machines, or just on nodes that will use clustering (i.e. it could be just in the Cluster documentation)